projectsmember/Pages/CooperationRequest/Profiles/HamidForoughnia.html

```diff
@@ -0,0 +1,165 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title> رزومه</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    
+    <style>
+        .material-symbols-outlined {
+            font-variation-settings:
+                    'FILL' 0,
+                    'wght' 400,
+                    'GRAD' 0,
+                    'opsz' 48
+        }
+    </style>
+
+</head>
+<body  dir="rtl">
+<div class="container border">
+    <div class="row">
+        <h1 class="mt-4">حمید فروغ نیا</h1>
+        <a href="https://dotnet.microsoft.com/en-us/languages/csharp">
+            <p>برنامه نویس c#</p>
+        </a>
+    </div>
+    <div class="row">
+        <div class="col">
+            <span class="material-symbols-outlined fs-6 fw-lighter">
+                <p style="display: inline-block;">
+                    استان قزوین، قزوین، منطقه ۳
+                </p>
+                location_on
+            </span>
+        </div>
+        <div class="col">
+            <span class="material-symbols-outlined fs-6 fw-lighter">
+                <p style="display: inline-block;">
+                    hamid.foroughnia@outlook.com
```

Comment:
Usually don't put your email out there as text, it is easy to be crawled and then again, getting spam emails. You can use services like https://www.generateit.net/email-to-image/ to generate an image of your email and then using that. (ideal case)